tests/topotato/run_userns.sh

```diff
@@ -17,7 +17,7 @@ if [ "$1" = "ns_inner" ]; then
 
 	ip link set lo up
 
-	exec pytest-3 "$@"
+	exec pytest "$@"
```

Comment:
Might be better to use `python3 -mpytest` instead.

Reply:
updated